WordPress/Classes/ViewRelated/NUX/Login.storyboard

```diff
@@ -134,78 +134,78 @@
                         <rect key="frame" x="0.0" y="0.0" width="383" height="676"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <stackView opaque="NO" contentMode="scaleToFill" verticalHuggingPriority="245" axis="vertical" spacing="10" translatesAutoresizingMaskIntoConstraints="NO" id="ZBL-pW-jTv" userLabel="Wrapper Stack View">
-                                <rect key="frame" x="0.0" y="64" width="383" height="612"/>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ozJ-hT-OEw">
```

Comment:
Ok, so all the stack views I had added are now just views 💔 I know that in particular I used transparent spacer viers to get the wpcom login screen to look right, but otherwise stack views are so nice! They allow a bunch of constraints to be implied instead of added explicitly, and adding, removing, and hiding views is much easier—tho that was a more important consideration before we decided to center each screen's primary text field (the error text could pop in/out nicely).

But beyond my whimpering over the loss of stack views, we've ended up with several levels of nested views, which I'm quite certain could be reduced down to just one, especially with the centered text field which all the form elements could layout relative to.